contrib/README.md

```diff
@@ -3,10 +3,9 @@ Repository Tools
 
 ### [Developer tools](/contrib/devtools) ###
 Specific tools for developers working on this repository.
-Contains the script `github-merge.py` for merging GitHub pull requests securely and signing them using GPG.
```

Comment:
Might make sense to mention the `maintainer-tools` repository (here, or somewhere else in a README), and that most developer tools live there.

Reply:
Agree, will update with a reference to the maintainer tools repo.

Reply:
Updated with a link to the maintainer-tools repository.